README.md

```diff
@@ -128,8 +128,7 @@ These dependencies are by-default available if you are using `projectmonai/monai
 
 ### [3D Slicer](https://download.slicer.org/) (radiology)
 
-Download **Preview Release** from https://download.slicer.org/ and install MONAI Label plugin from Slicer Extension
-Manager.
+Download and install 3D Slicer with the [installation page](https://docs.monai.io/projects/label/en/latest/installation.html) and install MONAI Label plugin from Slicer Extension Manager.
```

Comment:
sentence is not correct... may be you need to simply a bit

Reply:
Yup, have split the sentence into two sentences.